modules/ROOT/pages/navigation.adoc

```diff
@@ -1,19 +1,16 @@
 = Navigation between User Interfaces
-:description: == In Bonita Portal
-
-== In Bonita Portal
-
-The default Portal profile is User. From there, the user can navigate to their other profiles. +
-image:images/UI2021.1/navigation-portal.png[Navigation Portal]
-// {.img-responsive}
+:description: == In Bonita Applications
 
 == In Bonita Applications
 
 Users may be mapped with several profiles, opening permissions to access several applications.
 Or maybe one given profile is mapped with several applications.
-In both cases, the user can navigate between applications they have the permissions to access:
+In both cases, the user can navigate between applications they have the permissions to access.
 
+The first way to navigate to a different application is through the Bonita Layout.
 . The user clicks on the _mosaïc_ icon in the Application header
 . The user selects the tile of the application to navigate to
 image:images/UI2021.1/navigation-application.png[Navigation Application]
 // {.img-responsive}
+
+The second way is through the Application directory, which is the first application an user sees when connecting to Bonita: image:images/temp-release-notes-specific/new_app_directory_with_converted_custom_profile.png[Bonita Application Directory]
```

Comment:
an user sees when connecting to Bonita => a user sees when connecting to Bonita

Reply:
Added